carbon-design-system/ibm-security
# feat: upgrade Carbon to 10.16.0 release, upgrade Filter to Downshift v5 API

## Affected issues

Follow up to #678
Resolves #689 
Resolves #671

## Proposed changes

### Carbon

- upgrade Carbon packages to 10.16.0 release

### `Filter`

- upgrade to Downshift v5 API: https://github.com/downshift-js/downshift/blob/v5.2.1/README.md#usage

### `TagWallFilter`

- update `TagWallFilter` to render `Filter` contents via `body` prop of `TearsheetSmall` instead of `children` (note that `children` is deprecated in #678)

## Testing instructions

- ensure that `TagWallFilter` works as intended (due to Downshift upgrades)
- verify that components are stable after Carbon upgrade
